docs/k8s-quickstart.asciidoc

```diff
@@ -310,6 +305,7 @@ To aim for the best performance, the operator supports persistent volumes local
  * link:https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner[kubernetes-sigs local volume static provisioner] to setup static local volumes.
 
```

Comment:
Could we use this PR to backport the below change from master to 0.8 in the snippet above?

From:
```
storageClassName: gcePersistentDisk # can be any available storage class
```

To:
```
#storageClassName: standard # can be any available storage class
```

Reply:
Done.